internal/integration/helpers.go

```diff
@@ -130,6 +133,22 @@ func testServer(tb testing.TB) (*serverenv.ServerEnv, *http.Client) {
 	}
 
 	db := database.NewTestDatabase(tb)
+	if v := os.Getenv("DB_NAME"); v != "" && !testing.Short() {
```

Comment:
Why is this needed? I think the database util already sets up the secret manager and stuff, no?

Reply:
https://github.com/google/exposure-notifications-server/blob/2dfbe2f7cee56b5f0d6b5b66911d03f2ab29de1c/internal/database/database_util.go#L48
I don't think database_util.go sets up secret manager, not database from environment either